slemicro/delete_vm.sh

```diff
@@ -19,12 +19,14 @@ if [ $(uname -o) == "Darwin" ]; then
 	OUTPUT=$(
 	osascript <<-END
 	tell application "UTM"
-		set vm to virtual machine named "${VMNAME}"
-		if status of vm is started then stop vm
-		repeat
-			if status of vm is stopped then exit repeat
-		end repeat
-		delete virtual machine named "${VMNAME}"
+		if exists virtual machine named "SLEMicro" then
```

Comment:
Virtual machine name shouldn't be hardcoded.

Reply:
Somehow I am failing to force push the update. On it.